File: DSA basic q for beg/countvowelsinstr.py
vowels =  ['a','e','i','o','u']

def countvowels(str):
    count=0
    for i in range(len(str)):
        for j in range(len(vowels)):
            if str[i] == vowels[j]:
                count+=1

    return count

str = 'abc de'
print(countvowels(str)) 

File: DSA basic q for beg/java dsa beginers/Countvowelsinstr.java
public class Countvowelsinstr {
    char[] vowels = { 'a', 'e', 'i', 'o', 'u' };

    int countvowels(String str) {
        int count = 0;
        for (int i = 0; i < str.length(); i++) {
            char ch = Character.toLowerCase(str.charAt(i));
            for (char vowel : vowels) {
                if (vowel == ch){
                    count++;
                    break;
            }

        }}
        return count;

    }

    public static void main(String[] args) {
        String str = "abc deu";
        Countvowelsinstr cnt = new Countvowelsinstr();
        int result = cnt.countvowels(str);
        System.out.println("vowels are : " + result);
    }
}
